.env.example

```
@@ -7,9 +7,13 @@
 # The database URL is used to connect to your Neon database.
 POSTGRES_URL="postgresql://[USERNAME]:[PASSWORD]@[xyz]-pooler.c-2.us-east-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require"
 
+# Salt for hashing IP addresses in analytics (required in production)
+# Generate a secure random string, e.g., using `openssl rand -base64 32`
+HASH_SALT="change-me-to-a-secure-random-string"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)